### ISSUE TITLE
Feature Request:  Playlist hotkeys & another enhancement.

### ISSUE THREAD

**Issue by qwksilver** [FredTungsten/ScriptPlayer#171]
While video is loaded and playing 

- [ ] a: Hotkey to Move file 
- [ ] b: Hotkey to Move file (and remove from playlist)(and advance to next video)
- [ ] c: Hotkey to Delete to RecycleBin (and remove from playlist)(and advance to next video)
- [ ] d: Hotkey to remove from playlist (and advance to next video)

When right click, move (or move&remove), is chosen it doesn't remember to last location used as a destination.  It always reverts to the location of the file, but last destination would be more efficient.

It would be nice if the above hotkey move commands, used a default location set up in settings under paths.
It would also be great to have the hotkeys that remove from playlist also immediately skip to next video after you hit the hotkey.

i would really like to see the hotkey work without prompt (unless the default location is blank)  (ie corrupt video, missing script,  totally crap content, hit M and next video starts and the moved item is tucked away to be fixed later :)

same thing with delete recycle bin, no prompts just hit D crap content gone and on to the next.

mostly this is for the sake of using a remote control, the mouse and right clicking is rarely feasible during session.

**Comment by qwksilver** [comment]
#149 related to item c above
I was wondering if these shortcut keys would be coming soon, I've been doing a lot of library editing and one push deletes, and one button open move dialog would be glorious, i can script the destination and finish hitting ok externally with AutoHotKey.  It would also allow me to put the move and delete functions on my deckboard (streamdeck clone from old phone).

Thanks @FredTungsten and as always I love the player so much.

**Comment by FredTungsten** [comment]
I was planning to add some favorite folders to the move function. Adding a "default" for those might make quick work of covering most cases.

>I was wondering if these shortcut keys would be coming soon

I don't wanna make any promises, haven't been in the mood to add features for a while now - maybe when this whole pandemic is over/better and my general motivation to work on SP again returns.

**Comment by qwksilver** [comment]
> I don't wanna make any promises, haven't been in the mood to add features for a while now - maybe when this whole pandemic is over/better and my general motivation to work on SP again returns.

Fair enough, I used to be always out and doing stuff prepandemic, being stuck inside has had me working on all the things I had lacked the motivation on.  It's like, "why waste the time," for me.  It's interesting to observe how differently folks have reacted to the pandemic over all.

Well, Cheers, and here's hopping you find a second wind.

Ps.  if there are any other projects you have that you could use any help on, feel free to reach out, I do a bit of hardware electronics, and some 3dprinting and cad design.  (qwksilver@iname.com)  I may not have the funds to offer dontations, but I do have quite a bit of time to trade.

**Comment by FredTungsten** [comment]
Added some "Move To" shortcuts (bottom of the list) and a new dialog to choose folders from.

**Comment by qwksilver** [comment]
I like them the delete works great.  The move being available is good, but still throws the error I detailed in a bug topic #180.

**Comment by qwksilver** [comment]
this is more or less addressed and can be closed i think.
